src/app/style-panel/style-panel.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, AfterViewInit, ViewChildren } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Store } from '@ngrx/store';
+import { getFields, getGeneralStyle } from '../core/store/index';
+import { StyleServices } from '../shared/style.services';
+import { getStyle } from '../core/store/index';
+import {
+  ChangeStyleAction,
+  RemoveStyleAction,
+} from '../core/store/styles-fields/styleFields.actions';
+import { ChangeStyleGeneralAction } from '../core/store/styles-general/general-style.actions';
+import { valueDefault } from '../shared/value.sheets';
+import { FormArray, FormControl, FormGroup, Validators } from '@angular/forms';
+import { Observable, Observer } from 'rxjs';
+
+enum Attribute {
+  Placeholder = 'placeholder',
+  Required = 'required',
+  SelectValue = 'selectvalue',
+  SelectName = 'selectname'
+}
+
+@Component({
+  selector: 'app-style-panel',
+  templateUrl: './style-panel.component.html',
+  styleUrls: ['./style-panel.component.scss'],
+})
+export class StylePanelComponent implements AfterViewInit {
+  panelOpenState = false;
```

Comment:
false(undefined) by default
```suggestion
  panelOpenState: boolean;
```